fix: insert Youtube link (#862)
aziontech/docs: src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-astro.mdx

Wait a few minutes so the propagation takes place, and then access your application using the provided domain, which should be similar to `https://xxxxxxx.map.azionedge.net`.

> Watch a video guide about build with Astro on Azion’s YouTube channel:

Reviewer: ```suggestion
Watch a video guide about building with Astro on Azion’s YouTube channel:
```

Reviewer 2: ```suggestion
Watch an Astro building guide on Azion's YouTube channel:
```
